salsa_dancing_molecules/cli/main.py

```diff
@@ -51,6 +51,13 @@ def main():
                                   'simulation data. The file will be named '
                                   '$output_file-$formula.traj and -.csv '
                                   'for each material'))
+
+    nve_parser.add_argument('--potential',
+                            help=('Potential from taken from OpenKIM '
+                                  'database. If no potential is given a '
+                                  'standard Lennard-Jones potential '
+                                  'will be used'), default='')
```

Comment:
should'nt the default then be 'Lennard-Jones' and not ''

Reply:
Fixed!